Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'rails', '~> 5.0.7'
 gem 'rubocop'
 gem 'sassc-rails'
 gem 'spring'
+gem 'strscan'
```

Comment:
Dunno. It's a default gem, and I suddenly got a message about how it wasn't in the Gemfile.